app/views/idv/shared/_document_capture.html.erb

```diff
@@ -133,4 +133,3 @@
   window.LoginGov.assets = <%= raw asset_keys.map { |key| [key, asset_path(key)] }.to_h.to_json %>;
 <% end %>
 <%= javascript_packs_tag_once 'document-capture' %>
-<%= stylesheet_packs_with_chunks_tag 'document-capture' %>
```

Comment:
This will conflict with the `stages/prod` branch because we switched from `stylesheet_pack_tag ` to `stylesheet_packs_with_chunks_tag` in #4553, merged this morning.

Reply:
See branch `aduth-remove-webpacker-sass-prod` based on `stages/prod`, specifically can cherry-pick 99b26f1 onto `stages/prod`.